types/fingerpose/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "@definitelytyped/dtslint/dt.json" }
```

Comment:
You should remove this file; it is no longer required.

Reply:
Fixed.